src/main/java/services/notification_sending/EmailNotificationPresenter.java

```diff
@@ -0,0 +1,81 @@
+package services.notification_sending;
+
+import java.util.Properties;
+import javax.mail.*;
+import javax.mail.internet.*;
+
+/**
+ * Sends out email notifications.
+ * Keeps a record of whether the user enabled the email notification settings.
+ */
+public class EmailNotificationPresenter implements NotificationPresenter, SettingsRegistry {
+    private boolean enabled;
+    private String senderEmail;
+    private String password;
+    private String userEmail;
+    private String subject;
+
+    EmailNotificationPresenter(boolean enabled, String senderEmail, String password, String userEmail, String subject) {
+        this.enabled = enabled;
+        this.senderEmail = senderEmail;
+        this.password = password;
+        this.userEmail = userEmail;
+        this.subject = subject;
+    }
+
+    @Override
+    public void presentNotification(String message) {
+        //Get properties object
+        Properties props = new Properties();
+        props.put("mail.smtp.host", "smtp.gmail.com");
+        props.put("mail.smtp.socketFactory.port", "465");
+        props.put("mail.smtp.socketFactory.class",
+                "javax.net.ssl.SSLSocketFactory");
+        props.put("mail.smtp.auth", "true");
+        props.put("mail.smtp.port", "465");
+
+        //get Session
+        Session session = Session.getDefaultInstance(props,
+                new javax.mail.Authenticator() {
+                    protected PasswordAuthentication getPasswordAuthentication() {
+                        return new PasswordAuthentication(senderEmail, password);
+                    }
+                });
+
+        //compose message
```

Comment:
I'm being a little harsh on comments today (sorry!) This comment isn't quite true since the following block both composes the message and also sends it. It doesn't contribute much to readability either as the following lines are pretty straightforward. In a sense it's actually a little confusing since the end of the method has been documented as 'composing the message' instead of 'sending the message.' I don't think I'd really want a `// compose and send message` comment either since as just mentioned this block is fairly straight forward already.